packages/flutter_tools/lib/src/commands/update_packages.dart

```diff
@@ -59,6 +59,11 @@ class UpdatePackagesCommand extends FlutterCommand {
         help: 'Prints the dependency graph that is the transitive closure of '
               'packages the Flutter SDK depends on.',
         defaultsTo: false,
+      )
+      ..addFlag(
+        'verify-only',
```

Comment:
call this from the test script? (`/dev/bots/test.dart`)

Reply:
Done